packages/manager/apps/container/vite.config.js

```diff
@@ -69,6 +69,9 @@ module.exports = defineConfig({
       '@': path.resolve(path.join(__dirname, 'src')),
     },
   },
+  define: {
+    __VERSION__: process.env.VERSION ? `'${process.env.VERSION}'` : 'null',
```

Comment:
```suggestion
    __VERSION__: `process?.env?.VERSION?.toString()'`,
```
 ?
 
 If it's not set, it means it's already `null`, so it will return `undefined`